python/src/aiconfig/eval/metrics.py

```diff
@@ -261,37 +304,37 @@ def make_openai_structured_llm_metric(
             raise ValueError(f"Error making metric: {e}")
 
 
-def substring_match(substring: str, case_sensitive: bool = True) -> Metric[str, bool]:
-    async def _fn(datum: str) -> bool:
-        return _check_substring(
-            output_datum=datum,
-            substring=substring,
-            case_sensitive=case_sensitive,
-        )
+# 2. literal metrics
 
-    return Metric(
-        evaluation_fn=_fn,
-        metric_metadata=common.EvaluationMetricMetadata(
-            name="substring_match",
-            description="True (pass) if contains given substring",
-            best_value=True,
-            worst_value=False,
-            extra_metadata=dict(substring=substring, case_sensitive=case_sensitive),
-        ),
-    )
 
+@metric(
+    #
```

Comment:
nit: remove

Reply:
I'm going to just make the line length shorter. cc @Ankush-lastmile